debian/vyos-1x.install

```diff
@@ -8,3 +8,13 @@ etc/vyos
 lib/
 opt/
 usr/
+usr/lib/
```

Comment:
Why is this needed? It's already covered by `usr/`

Reply:
I would like to ask this exact question to the Debian developers as testing showed that without being explicit the files would not be included. Hence this exact question in the PR comment as I am also puzzled and wondered if there was some syntax I am not aware of to make this list shorter!

This is the smallest subset which worked and included all the files (comparing the content with the previously installed package and the new one using `dpkg -l` and `diff` ..